### PR TITLE
treewide: Replace xhtml content type with html, remove xhtml workaround, replace self closing tags

### PR DESCRIPTION
**Pull request by MyIgel**
As a follow up to #2398 / bbc00017
* Replace xhtml content type with html -> pages are interpreted as HTML5 which is the current standard and much more flexible
* Remove old xhtml workaround which is not needed anymore
* Updated `prepare_content` docs accordingly

### CODE REVIEW

**Review by skorpy2009 — APPROVED** [review]

File: docs/releases/v2016.1.rst [hunk 1]
```diff
@@ -236,7 +236,7 @@ Site changes
     "</p>"
     "<div class=\"the-key\">"
     " # <%= hostname %>"
-    " <br/>"
+    " <br>"
```
Comment (NeoRaider) [inline docs/releases/v2016.1.rst:239]: Please don't change old release notes.

**Review by NeoRaider — CHANGES_REQUESTED** [review]

File: docs/releases/v2016.1.rst (hunk 1 above)
Comment (MyIgel) [reply docs/releases/v2016.1.rst]: Ärm jeah, thats a great idea & fixed :D

**Review by NeoRaider — APPROVED** [review]

**Comment by rubo77** [comment]
Nice.

Why was the XHTML tag there in the first place?

**Comment by MyIgel** [comment]
I would guess legacy reasons? Meaning: It was required for some browsers to work back then and is now totally obsolete

**Comment by NeoRaider** [comment]
Yeah, the code originated from LuCI, which was written before HTML5 existed. Back then XHTML was considered the future of HTML I guess (and personally I still prefer XML syntax even with HTML5 for many usecases - but for gluon-web it causes more problems than it solves).

**Pull request merged**